d2l-icon.html

```diff
@@ -105,7 +111,7 @@
 				this.async(function() {
 					this.__initialize();
 					this.updateStyles(); // need to re-calc iron-icon variables
-				}, 1);
```

Comment:
Interesting.  That seems odd that they'd implement different behavior merely as a result of 0 vs 1ms wait time.  Nevertheless, I don't mind putting this back.